### DISCUSSION TITLE
Expo 开发 Airbnb Clone 全栈应用 | Alan Wang

### DISCUSSION THREAD

**Discussion by giscus** [3Alan/site#47] [Announcements]
# Expo 开发 Airbnb Clone 全栈应用 | Alan Wang

使用 Expo 开发 Airbnb Clone 全栈应用，使用 MongoDB 作为数据库，使用 React Query 作为数据管理工具，使用 Prisma 作为 ORM。

https://www.alanwang.site/blog/expo-airbnb-clone

**Comment by ShikinChen** [comment]
能否开源mongo的数据库的数据和api服务 我看github的项目只有数据库结构和简单的api服务而且docker-compose.yml配置有问题 不知道为什么我不能使用replSet后面我删除才能使用

**Reply by 3Alan** [reply]
所有的api都在这个[目录](https://github.com/3Alan/airbnb-clone/tree/main/src/app/api)下，使用的是Expo的API Routes，数据我后面有空导出一份
